core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.jelly

```diff
@@ -26,6 +26,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 <l:layout title="${%Manage Old Data}">
   <st:include page="sidepanel.jelly" it="${app}"/>
+  <l:breadcrumb title="${%Manage Old Data}" />
```

Comment:
❌ This one should be removed

To test, the easiest path I found, you have to open the Script Console, execute the following code:
```
hudson.diagnosis.OldDataMonitor.report(Jenkins.get(), "1.0")
```

Then, either click on the "Manage" button from that admin monitor message, or directly to http://localhost:8080/jenkins/administrativeMonitor/OldData/manage.

<details><summary>Gif</summary>

![manage_old_data](https://user-images.githubusercontent.com/2662497/204617242-acad6533-e5cd-4c4d-ad21-5223a1e5e12c.gif)


</details>